SPECS/azurelinux-rpm-macros/macros.python-srpm

```diff
@@ -1,28 +1,95 @@
+# There are multiple Python 3 versions packaged, but only one can be the "main" version
```

Comment:
I think it might be safe to also update other files we have in this spec, which come from Fedora's `python-rpm-macros` spec. Otherwise, there's a greater risk of hitting some obscure issues stemming from the fact, that some of these macro files and script are from an older version of Fedora, than this one.

Reply:
What do you feel about making those additional changes in a follow-up PR? I tried to limit this PR to python-related stuff and it's already a pretty large PR

Reply:
I've pushed updates to all of these files